.github/workflows/auto-update.yml

```
@@ -0,0 +1,19 @@
+name: Update branch marked auto merge
+
+on:
+  # Runs every 15 minutes
+  schedule:
+    - cron: '*/15 * * * *'
```

```suggestion
    - cron: '*/15 7-19 * * 1-5'
```
At every 15th minute past every hour from 7 through 19 on every day-of-week from Monday through Friday. (https://crontab.guru/#*/15_7-19_*_*_1-5)